tools/src/code-review/reviewers/checkMissingChangelogs.ts

```diff
@@ -22,7 +22,8 @@ export default async function ({ pullRequest, diff }: ReviewInput): Promise<Revi
   }
 
   const changelogLinks = pkgsWithoutChangelogChanges
-    .map((pkg) => `- ${relativeChangelogPath(pullRequest, pkg)}`)
+    .map((pkg) => pullRequest.head && `- ${relativeChangelogPath(pullRequest.head, pkg)}`)
```

Comment:
When is it falsy? In that case the array would be empty but the result would still say that there are missing links. It's probably better to put it as additional condition to the `if` above (like you did in `reviewForbiddenFiles.ts`).

Reply:
We are using the custom modified types there, but they are still generated from the Octokit REST methods, so it's hard to tell when exactly `head` in not provided in the response:
* https://github.com/expo/expo/blob/main/tools/src/GitHub.ts#L266;L269